diff --git a/dist/index.js b/dist/index.js
index 68bc0ef7e2084710f7d6159d256d482d5130509e..9964e5fbdfd27236b1a657810d039648f72cfc69 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1261,7 +1261,7 @@ var LLMGatewayChatCompletionBaseResponseSchema = import_v45.z.object({
       reasoning_tokens: import_v45.z.number()
     }).nullish(),
     total_tokens: import_v45.z.number(),
-    cost: import_v45.z.number().optional(),
+    cost: import_v45.z.any().optional(),
     cost_details: import_v45.z.object({
       upstream_inference_cost: import_v45.z.number().nullish()
     }).nullish()
@@ -2044,7 +2044,7 @@ var LLMGatewayCompletionChunkSchema = import_v46.z.union([
         reasoning_tokens: import_v46.z.number()
       }).nullish(),
       total_tokens: import_v46.z.number(),
-      cost: import_v46.z.number().optional()
+      cost: import_v46.z.any().optional()
     }).nullish()
   }),
   LLMGatewayErrorResponseSchema
diff --git a/dist/index.mjs b/dist/index.mjs
index 571cc28ac5117a98c0b35d14beab164a9b41f986..25506da37de70aec726363041ac24805edfb9302 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1226,7 +1226,7 @@ var LLMGatewayChatCompletionBaseResponseSchema = z6.object({
       reasoning_tokens: z6.number()
     }).nullish(),
     total_tokens: z6.number(),
-    cost: z6.number().optional(),
+    cost: z6.any().optional(),
     cost_details: z6.object({
       upstream_inference_cost: z6.number().nullish()
     }).nullish()
@@ -2009,7 +2009,7 @@ var LLMGatewayCompletionChunkSchema = z7.union([
         reasoning_tokens: z7.number()
       }).nullish(),
       total_tokens: z7.number(),
-      cost: z7.number().optional()
+      cost: z7.any().optional()
     }).nullish()
   }),
   LLMGatewayErrorResponseSchema
diff --git a/dist/internal/index.js b/dist/internal/index.js
index cd2a9b9d5538e26d6397b5c78d053cb35f5c8ef0..1c08400c8ecb1b7e421cb82281cebd41c3d4495b 100644
--- a/dist/internal/index.js
+++ b/dist/internal/index.js
@@ -1216,7 +1216,7 @@ var LLMGatewayChatCompletionBaseResponseSchema = import_v45.z.object({
       reasoning_tokens: import_v45.z.number()
     }).nullish(),
     total_tokens: import_v45.z.number(),
-    cost: import_v45.z.number().optional(),
+    cost: import_v45.z.any().optional(),
     cost_details: import_v45.z.object({
       upstream_inference_cost: import_v45.z.number().nullish()
     }).nullish()
@@ -1999,7 +1999,7 @@ var LLMGatewayCompletionChunkSchema = import_v46.z.union([
         reasoning_tokens: import_v46.z.number()
       }).nullish(),
       total_tokens: import_v46.z.number(),
-      cost: import_v46.z.number().optional()
+      cost: import_v46.z.any().optional()
     }).nullish()
   }),
   LLMGatewayErrorResponseSchema
diff --git a/dist/internal/index.mjs b/dist/internal/index.mjs
index dda06e1accb954999d1782af5288472c1db9b1d2..58662fd1f4c6788b8964c38ad01a064590c21779 100644
--- a/dist/internal/index.mjs
+++ b/dist/internal/index.mjs
@@ -1181,7 +1181,7 @@ var LLMGatewayChatCompletionBaseResponseSchema = z6.object({
       reasoning_tokens: z6.number()
     }).nullish(),
     total_tokens: z6.number(),
-    cost: z6.number().optional(),
+    cost: z6.any().optional(),
     cost_details: z6.object({
       upstream_inference_cost: z6.number().nullish()
     }).nullish()
@@ -1964,7 +1964,7 @@ var LLMGatewayCompletionChunkSchema = z7.union([
         reasoning_tokens: z7.number()
       }).nullish(),
       total_tokens: z7.number(),
-      cost: z7.number().optional()
+      cost: z7.any().optional()
     }).nullish()
   }),
   LLMGatewayErrorResponseSchema
